styles/main.scss

```diff
@@ -23,6 +25,8 @@ body {
 
 h1 {
   font-size: 40px;
+  text-shadow: 0px 1px 2px rgba(0, 0, 0, 0.2);
```

Comment:
This color is used multiple times for text-shadow, could you please refactor to variable?